activesupport/lib/active_support/core_ext/hash/conversions.rb

```diff
@@ -68,7 +68,7 @@ class Hash
   #
   # By default the root node is "hash", but that's configurable via the <tt>:root</tt> option.
   #
-  # The default XML builder is a fresh instance of <tt>Builder::XmlMarkup</tt>. You can
+  # The default XML Builder is a fresh instance of <tt>Builder::XmlMarkup</tt>. You can
```

Comment:
I think this refers to _a_ XML builder, not the XML Builder template format or the class to delegate to. We also don't upcase `builder` in the following lines. So I think we should keep this as is.